src/Components/Blazor/Build/src/Microsoft.AspNetCore.Blazor.Build.csproj

```diff
@@ -26,7 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Condition="'$(BuildNodeJS)' != 'false'" Include="$(RepoRoot)src\Components\Web.JS\Microsoft.AspNetCore.Components.Web.JS.npmproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Condition="'$(BuildNodeJS)' != 'false' and '$(BuildingInsideVisualStudio)' != 'true'" Include="$(RepoRoot)src\Components\Web.JS\Microsoft.AspNetCore.Components.Web.JS.npmproj" ReferenceOutputAssembly="false" />
```

Comment:
Is this to make the build faster inside VS? Does it prevent people from changing the TS code and then building in VS without also building on the command line?

Reply:
There's a problem in general right now, as explained in the commit: https://github.com/aspnet/AspNetCore/pull/12654/commits/61ef8fc497896723c26bd1f1037d2b7a86fdca43

This isn't needed anymore, since the change that caused it got reverted.